ast/src/main/java/com/graphicsfuzz/common/tool/PrettyPrinterVisitor.java

```diff
@@ -94,6 +94,7 @@
   private boolean inFunctionDefinition = false;
   private final boolean emitGraphicsFuzzDefines;
   private final Optional<String> license;
+  private boolean insideForStatement = false;
```

Comment:
Please rename to `insideForStatementHeader` (in relation to a comment below), and on the line above have a brief comment to explain that this allows us to format declarations differently if they are part of the header of a for statement.

Reply:
Done.